pandas/tests/frame/test_indexing.py

```diff
@@ -1093,10 +1127,11 @@ def test_fancy_index_int_labels_exceptions(self):
                    r" dtype='object'\)\] are in the \[index\]")
             with pytest.raises(KeyError, match=msg):
                 self.frame.ix[['foo', 'bar', 'baz']] = 1
-            msg = (r"None of \[Index\(\['E'\], dtype='object'\)\] are in the"
-                   r" \[columns\]")
-            with pytest.raises(KeyError, match=msg):
-                self.frame.ix[:, ['E']] = 1
+            # partial setting now allows this GH13658
```

Comment:
don;t comment out, simply remove

Reply:
I commented out the test following the test immediately below on line 1136. Should it be removed too?

Reply:
yes try that

Reply:
> yes try that

Yes for this question too?

> @jreback can you please confirm if I should just fix this issue by inserting columns filled with `NaN` first?